python/common/org/python/stdlib/datetime/Date.java

```diff
@@ -0,0 +1,304 @@
+package org.python.stdlib.datetime;
+
+import java.util.Collections;
+
+public class Date extends org.python.types.Object {
+
+    @org.python.Attribute
+    public org.python.Object year = __year__();
+
+    @org.python.Attribute
+    public org.python.Object month = __month__();
+
+    @org.python.Attribute
+    public org.python.Object day = __day__();
+
+    @org.python.Attribute
+    public static final org.python.Object min = __min__();
+
+    @org.python.Attribute
+    public static final org.python.Object max = __max__();
+
+    @org.python.Method(__doc__ = "")
+    public Date(org.python.Object[] args, java.util.Map<java.lang.String, org.python.Object> kwargs) {
+
+	super();
+
+	if (args.length + kwargs.size() > 3) {
+	    int val = args.length + kwargs.size();
+	    throw new org.python.exceptions.TypeError("function takes at most 3 arguments (" + val + " given)");
+	}
+
+	if (args.length + kwargs.size() == 3) {
+	    if (kwargs.get("year") != null && args.length == 2) {
+		throw new org.python.exceptions.SyntaxError("positional argument follows keyword argument");
```

Comment:
`date(1, 2, year=2)` should return `TypeError: function missing required argument 'day' (pos 3)`, same for month.